modules/Users/Users.php

```diff
@@ -527,11 +527,36 @@ public function change_password($user_password, $new_password, $dieOnError = tru
 			$this->error_string = $mod_strings['ERR_PASSWORD_INCORRECT_OLD'];
 			return false;
 		}
-
 		//set new password
 		$crypt_type = $this->DEFAULT_PASSWORD_CRYPT_TYPE;
 		$encrypted_new_password = $this->encrypt_password($new_password, $crypt_type);
 
+		global $adb;
+		require_once 'modules/Emails/Emails.php';
+
+		$emailObj = new Emails;
```

Comment:
eliminate this line, the method is static so you don't need to waste time instantiating the module